input/fsh/Aliases.fsh

```diff
@@ -1,7 +1,7 @@
 Alias: $StatusCodes = http://hl7.org/fhir/encounter-status
 Alias: $ActCodes = http://terminology.hl7.org/CodeSystem/v3-ActCode 
 Alias: $EventStatus = http://hl7.org/fhir/event-status
-Alias: $CategoryCodes = http://medcomfhir.dk/fhir/carecommunication/CodeSystem/medcom-careCommunication-categoryCodes
+Alias: $CategoryCodes = http://medcomfhir.dk/ig/carecommunication/CodeSystem/medcom-careCommunication-categoryCodes
```

Comment:
Er det korrekt at den er ændret til ig, når de andres steder er ændret til FHIR? :) Jeg kan ikke lige se hvad der er korrekt

Reply:
medcomfhir.dk/ig/. Men det skal være 'terminology' i stedet for 'carecommunication'. det retter jeg lige.